make-up/code/5305-dea-bibovska/src/app/models/mapping.ts

```diff
@@ -0,0 +1,54 @@
+import { Recipie, Ingredients, Ingredient, Cuisine } from "./client";
```

Comment:
There is a bit inconsistency between the mappings here and the usage in the data-service, but still +5 for the mapping